gateway/1.4/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc

```diff
@@ -255,3 +256,62 @@ spec:
 ----
 
 //end::splunk-configuration-example[]
+
+//tag::log-output-examples[]
+
+== Logging Event Examples
+The following log output examples demonstrate possible outputs that may occur for the following events:
+
+* <<graceful-shutdowns, Graceful shutdowns>>
```

Comment:
To avoid issues in the future, ensure that all anchors have their corresponding anchor IDs.